### ISSUE TITLE
radiff2 code, usage message and man page are inconsistent

### ISSUE THREAD

**Issue by aamcintosh** [radareorg/radare2#18244]
## Environment
(Doesn't really matter, but for what it's worth)
```sh
$ date
Mon Jan 18 21:01:09 EST 2021
$ r2 -v
radare2 4.5.1 6 @ linux-x86-64 git.4.5.1
commit: b148fa3e1cbf0d4e4d1d279af90e30520669e397 build: 2021-01-15__08:02:23
$ uname -ms
Linux x86_64
```

## Description
```
libr/main/radiff2.c
line 427:
    printf ("Usage: radiff2 [-abBcCdjrspOxuUvV] [-A[A]] [-g sym] [-m graph_mode][-t %%] [file] [file]\n");
line 445:
    "  -O         code diffing with opcode bytes only\n"
line 963:
    r_getopt_init (&opt, argc, argv, "Aa:b:BCDe:npg:m:G:OijrhcdsS:uUvVxXt:zqZ");
Line 445 agrees with the code , but in man/radiff2.1, lines 44-45:
.It Fl O
Do code diffing with all bytes instead of just the fixed opcode bytes
```
The concise version of the usage message is missing a few options and is vague about which flags require arguments.

## Test

Not much testing is required.  Compare the output of `radiff2 -h` and `man radiff2` for the `-O` inconsistency, though the problem is apparent by inspection of the source.  The concise version of `radiff2 -h` is missing the `-q` option, for example.  The following brings the concise usage message in line with the `getopt` call:
```
printf ("Usage: radiff2 [-abBcCdeGhijnrOpqsSxuUvVzZ] [-A[A]] [-g sym] [-m graph_mode][-t %%] [file] [file]\n");
```
(The gratuitous reordering just makes it harder for the two to get out of sync again.)

**Comment by trufae** [comment]
Maybe we should generate the manpages from source, but keeping short, long help and this in sync doesnt seems easy to me, one option would be to use spp to preprocess a template of the manpage with the relevant details of the tool and fill the rest with r2 -hsomething to generate the help message in manpage format. Additional sections like examples and description can be in the template. We will still need to maintain a duplicated list od help because the getopt is not in sync with the help message which is just plain text. So the overall thing ends up in two options:

- fix the manpage by hand
- Write a getopt frontend that parses arguments and generates help message and manpages.

first option sounds more ressonable in the short term to me

**Comment by aamcintosh** [comment]
No argument here.  I gave you a PR with a fix-by-hand.  Apply it (or maybe you have already) and close this.

**Comment by trufae** [comment]
uhm wasnt this fixed in 5a207ff69c1d3bd262351995ccf098f36156eb9f ?
